Tests/LoggingTests/LoggingTest.swift

```diff
@@ -134,13 +142,14 @@ class LoggingTest: XCTestCase {
         let testLogging = TestLogging()
         Logging.bootstrap(testLogging.make)
         var logger = Logging.make("\(#function)")
-        logger.logLevel = .error
+        logger.logLevel = .fault
 
         logger.debug(self.dontEvaluateThisString(), metadata: ["foo": "\(self.dontEvaluateThisString())"])
         logger.trace(self.dontEvaluateThisString())
         logger.info(self.dontEvaluateThisString())
         logger.warning(self.dontEvaluateThisString())
         logger.log(level: .warning, message: self.dontEvaluateThisString())
+        logger.error(self.dontEvaluateThisString())
```

Comment:
Add fault and reorder a little bit here perhaps?

```swift
        logger.warning(self.dontEvaluateThisString())
        logger.error(self.dontEvaluateThisString())
        logger.fault(self.dontEvaluateThisString())
        logger.log(level: .warning, message: self.dontEvaluateThisString())
```

Reply:
The purpose of the test is to verify that autoclosures are not evaluated for disabled log levels.  If we added a call to `logger.fault()` the test would fail because the log message would be enabled so the message would be evaluated.

Reply:
Ah, I missed the `        logger.logLevel = .fault` line, all good then :)